gm4_boots_of_ostara/data/gm4_boots_of_ostara/function/main.mcfunction

```diff
@@ -1,4 +1,3 @@
-execute as @a[gamemode=!spectator,gamemode=!adventure,predicate=gm4_boots_of_ostara:boots_equipped] at @s run function gm4_boots_of_ostara:player_convert
-execute as @e[type=armor_stand,predicate=gm4_boots_of_ostara:boots_equipped] at @s run function gm4_boots_of_ostara:armor_stand_convert
+execute as @e[gamemode=!spectator,gamemode=!adventure,type=#gm4_boots_of_ostara:valid_boot_wearers,predicate=gm4_boots_of_ostara:boots_equipped] at @s run function gm4_boots_of_ostara:convert
```

Comment:
The reason why the tests fail is that by using `gamemode=!spectator` the game will automatically restrict the selector to players. You will probably need two separate commands in this file.

Reply:
Resolved with b4f757d